wiki/osu!supporter/en.md

```diff
@@ -1,41 +1,39 @@
-![osu!supporter tag](Supporter tag.png "osu!supporter tag")
+# osu!supporter
 
-Introducing
-===========
+![osu!supporter tag](Supporter_tag.png "osu!supporter tag")
 
-<img src="Osu-support.png" title=" osu!supporter link on main page." alt=" osu!supporter link on main page." width="200" />
+## Introducing
```

Comment:
Remove this heading.
This declares that this is the _lead section_ but the first paragraph of all articles are always implied to be the _lead section_, so this is redundant and should be removed.
It also doesn't make much sense anyways.